library/lua/script-manager.lua

```diff
@@ -214,14 +214,44 @@ function get_active_mods()
     local ol = df.global.world.object_loader
 
     for idx=0,#ol.object_load_order_id-1 do
-        local path = ol.object_load_order_src_dir[idx]
+        local path = nil
+        if ol.object_load_order_src_dir[idx] then
+            path = ol.object_load_order_src_dir[idx].value
```

Comment:
`src_dir` is a `vector<path>` in 52.02 and so its elements do not have the `.value` metamethod. This code will not work with 52.02, or against develop